.github/workflows/package-audit-test-tools.yaml

```
@@ -0,0 +1,105 @@
+name: Package dev tools infrastructure
+
+on:
+  push:
+    paths:
+      - audit-test-tools/**
+  workflow_dispatch: {}
+
+env:
+  AWS_REGION: eu-west-2
+
+jobs:
+  validate-build:
+    name: Run tests and validation
+    permissions:
+      id-token: write
+      contents: read
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: audit-test-tools
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+      - name: Setup Python 3.8
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.8'
+      - name: Setup SAM CLI
+        uses: aws-actions/setup-sam@v2
+      - name: Assume AWS Validate role
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-region: ${{ env.AWS_REGION }}
+          role-to-assume: ${{ secrets.GH_ACTIONS_VALIDATE_ROLE_ARN_AUDIT_TEST_TOOLS }}
+      - name: Node - Enable Corepack
+        run: corepack enable
+      - name: Install ESBuild binaries
+        run: yarn unplug
+      - name: Run tests
+        if: always()
+        run: yarn test
+      - name: Build Lambda functions
+        if: always()
+        run: yarn build
+      - name: Validate SAM template
+        if: always()
+        run: sam validate
+      - name: Run Checkov on SAM template
+        if: always()
+        uses: bridgecrewio/checkov-action@master
+        with:
+          file: audit-test-tools/template.yaml
+          quiet: true
+          framework: cloudformation
+      - name: Upload build artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: audit-test-tools-sam-template
+          path: |
+            audit-test-tools/dist/
+            audit-test-tools/template.yaml
+
+  package:
+    strategy:
+      matrix:
+        target: [AUDIT_DEV]
```

You might also want to add AUDIT_BUILD as a target?

```suggestion
        target: [AUDIT_DEV, AUDIT_BUILD]
```

---

As discussed, we've found that this `target` property doesn't actually do anything (by looking at dev-tools and the lack of effect it has there!)